.github/workflows/linux_cuda_wheel.yaml

```diff
@@ -2,6 +2,11 @@ name: Build and test Linux CUDA wheels
 
 on:
   pull_request:
+    paths:
+      - src/torchcodec/*
+      - packaging/*
+      - test/*
+      - .github/workflows/linux_cuda_wheel.yaml # self reference
```

Comment:
We also run the benchmarks in that file, we may want to add the benchmark folder here?

https://github.com/pytorch/torchcodec/blob/f2a17fb7cb155839bcf5258168730c3807059482/.github/workflows/linux_cuda_wheel.yaml#L144